src/main/java/org/reactome/release/resourcechecker/FTPFileResourceChecker.java

```diff
@@ -0,0 +1,203 @@
+package org.reactome.release.resourcechecker;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.net.ftp.FTPClient;
+import org.apache.commons.net.ftp.FTPFile;
+
+import org.reactome.release.Resource;
+
+public class FTPFileResourceChecker implements FileResourceChecker {
+	private Resource resource;
+
+	private static final String DEFAULT_USER_NAME = "anonymous";
+
+	// It is common practice to give an e-mail for the "anonymous" password
+	// as a courtesy to the FTP server's site operators so they know who is
+	// accessing their service (https://stackoverflow.com/a/20031581)
+	private static final String DEFAULT_PASSWORD  = "help@reactome.org";
```

Comment:
Is this the email address the AddLinks process uses?

Reply:
I don't know.  Does the AddLinks process provide an FTP password if it uses anonymous access?

Reply:
I thought our Help E-mail was best if they wanted a Reactome e-mail to get in touch with us about accessing their FTP server.

Reply:
I can't recall, I'd have to look through the code. I think I just used "ANONYMOUS" or something like that.

Reply:
Okay, from what I've read, any value can be used for the password when logging on with a user of `anonymous`, but an e-mail is a courtesy.